README.md

```diff
@@ -38,7 +38,7 @@ For using any sensitive data/secrets like Azure Service Principal or MySQL Conne
 Follow the steps to configure the secret:
   * Define a new secret under your repository **Settings** > **Secrets** > **Add a new secret** menu
   * Paste the contents of the Secret (Example: Connection String) as Value
-  * Also, copy the connection string from Azure MySQL DB which is under **Connection strings > ADO.NET** and of the format: Server={your_server}; Port=3306; Database={your_database}; Uid={your_user}; Pwd={your_password}; SslMode=Preferred;
+  * Also, copy the connection string from Azure MySQL DB which is under **Connection strings > ADO.NET** and of the format: ```Server={your_server}; Port=3306; Database={your_database}; Uid={your_user}; Pwd={your_password}; SslMode=Preferred;```(Database is optional)
```

Comment:
looks fine @aksm-ms.. One lil change.. should we call out that the connection string needs to be copied from Azure portal?

Reply:
do we want to state that. Database is optional and a new DB can be created using scripts in case it doesnt exist?

Reply:
> looks fine @aksm-ms.. One lil change.. should we call out that the connection string needs to be copied from Azure portal?

since it is already mentioned as Azure Mysql DB, we do not have to state it.

Reply:
> do we want to state that. Database is optional and a new DB can be created using scripts in case it doesnt exist?

it need not be explicitly stated in my opinion because we can login to mysql without using a DB. in a case where user connects to database "A" and creates a new database "B" via scripts, it will not cause any errors.